### PR TITLE
loader/svg: Check invalid color

### PR DESCRIPTION
**Pull request by hermet**
Checks whether the string that can be specified in Color is valid.

example)
```
     style="stroke:asdasd"
```

related issue: https://github.com/thorvg/thorvg/issues/1255

### CODE REVIEW

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 1]
```diff
@@ -735,22 +738,24 @@ static void _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
                     tb /= 100.0f;
                     brightness = _skipSpace(brightness + 1, nullptr);
                     if (brightness && brightness[0] == ')' && brightness[1] == '\0') {
-                       if (_hslToRgb(th, ts, tb, r, g, b)) return;
+                       return _hslToRgb(th, ts, tb, r, g, b);
```
Comment (hermet) [inline src/loaders/svg/tvgSvgLoader.cpp:741]: @JSUYA please verify this

**Comment by hermet** [comment]
image

@JSUYA This svg has been broken by this patch. I think you should take care of this...

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 2]
```diff
@@ -713,6 +715,7 @@ static void _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
                 }
             }
         }
+        return true;
     } else if (ref && len >= 3 && !strncmp(str, "url", 3)) {
         if (*ref) free(*ref);
         *ref = _idFromUrl((const char*)(str + 3));
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:721]: missing `return true;`

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 3]
```diff
@@ -735,22 +738,24 @@ static void _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
                     tb /= 100.0f;
                     brightness = _skipSpace(brightness + 1, nullptr);
                     if (brightness && brightness[0] == ')' && brightness[1] == '\0') {
-                       if (_hslToRgb(th, ts, tb, r, g, b)) return;
+                       return _hslToRgb(th, ts, tb, r, g, b);
                     }
                 }
             }
         }
+        return true;
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgLoader.cpp:746]: please remove this.
If hsl color parsing is successful, it should return at line 741. If line 741 is not touched, _toColor() should be fail.

**Pull request merged**